test/test_overrides.py

```diff
@@ -318,12 +318,12 @@ def generate_tensor_like_torch_implementations():
     for namespace, funcs in get_overridable_functions().items():
         for func in funcs:
             if func not in testing_overrides:
-                untested_funcs.append("{}.{}".format(namespace, func.__name__))
+                untested_funcs.append("{}.{}".format(namespace.__name__, func.__name__))
     msg = (
-        "The following functions are not tested for __torch_function__ "
-        "support, please ensure there is an entry in the dict returned by "
-        "torch._overrides.get_testing_overrides for this function or if a "
-        "__torch_function__ override does not make sense, add an entry to "
+        "The following functions are not tested for __torch_function__ \n"
```

Comment:
Do we really need to add these newlines?

Reply:
well, we don't "really" need them of course. It's a cosmetic change to make the error look better, both in the terminal and in the Dr.CI logs:

![image](https://user-images.githubusercontent.com/344507/102788101-6a2c8e80-43a2-11eb-9aec-8a920c94aa31.png)

I'm happy to revert the change if you want

Reply:
Let's remove it from this PR since it seems like a change related to torch_function and not the implementation torch.linalg.svd.

Reply:
ok, reverted by a2e2781e0f